core/src/filed/filed.cc

```diff
@@ -66,6 +67,7 @@ static void usage()
         "        -g <group>  run as group <group>\n"
         "        -k          keep readall capabilities\n"
         "        -m          print kaboom output (for debugging)\n"
+        "        -p <path>   use <path> as destination when creating pidfile\n"
```

Comment:
we should probably `#if !defined(HAVE_WIN32)` this (in all daemons).
Also, I guess `<file>` is more appropriate, as otherwise users might think you need to pass a directory only.
Maybe we can just do something along the lines of
```suggestion
        "        -p <file>   create a pidfile (default: none)\n"
```